### DISCUSSION TITLE
Query with Filter on FlowField

### DISCUSSION THREAD

**Discussion by SKeiler** [StefanMaron/BusinessCentral.LinterCop#778] [New Rule]
Seem's to be a standard error, but currently (BC23 and BC24) the client crashes if a Enum-Value is used as filter in a query.

![image](https://github.com/user-attachments/assets/e9eab8db-adca-4cd7-a7ab-86dd850c7f95)

Also what's the best way to report the error to Microsoft?

**Comment by Arthurvdv** [comment]
This does indeed seems like unexpected behavior, especially if the client crashes.

The best way to report this issue is to submit it directly on the [Microsoft AL repository](https://github.com/microsoft/AL).

**Reply by SKeiler** [reply]
@Arthurvdv I tried this last time with an client crash and got the following anser "As a reminder, this repository only handles issues to do with the compiler or VSCode development experience for AL.".

But I will try it again...

**Reply by Arthurvdv** [reply]
You're right, I've pointed you to the wrong repository 😳

If you're stuck then the [D365BC Partner Community](https://www.yammer.com/dynamicsnavdev) could be a alternative?

**Reply by SKeiler** [reply]
Not really. About Yammer I tried such topics a few times, but every time there was no reaction (from somebody from Microsoft).

It's a little bit annoying, because it is a wrong behavior in the al language. The Github-topics are for the AL-Compiler and bugs/new things for the Base App, The MS Support reacts only if its reproducable in an cronus environment.
